blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+---
+**title:** New Tech Journey With RGSoC!
```

Comment:
Hi @StlMaris123 ! Thanks for your PR. There is no need of '**' in the parameters here. Please remove them.